ios/FluentUI.Demo/FluentUI.Demo/Demos/ThemingDemoController.swift

```diff
@@ -190,3 +205,22 @@ open class CustomStyle: FluentUIStyle {
         }
     }
 }
+
+class CustomCardNudgeTokens: CardNudgeTokens {
+    let purplePrimary: ColorSet = ColorSet(light: 0x6227A7)
+    override var accentColor: ColorSet {
```

Comment:
Why we don't use the TokenSet approach for the Control tokens in the instance based customization scenario as well?

Reply:
Good question! Because a `TokenSet` mandates a consistent token type across its entire set, and by their nature Control tokens can have a variety of different token types.

As discussed offline, there is room for iteration on the Control token pattern, but that will be out of scope for this PR.